packages/date-picker/src/FilterDateRangePicker/components/DateRangeInputField/DateRangeInputField.tsx

```diff
@@ -86,7 +99,9 @@ export const DateRangeInputField = React.forwardRef<
           <DateInput
             ref={inputRangeStartRef}
             id={`${id}--from`}
-            aria-describedby={inputDescribedBy}
+            aria-describedby={dateStartInputDescribedBy}
+            aria-errormessage={dateStartErrorMessageId}
+            aria-invalid={dateStartErrorMessageId ? true : false}
```

Comment:
Small tip:
This is a bit "magical" here, without much context I must assume reading this that the presence of a error message ID means that the field is invalid.
To make this more readable you could direct this into a variable and use that instead like so:
```
const isInvalid = dateStartErrorMessageId !== undefined;
...
aria-invalid={isInvalid}
```